docs/tutorials/ios_merging.ipynb

```diff
@@ -32,7 +32,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
```

Comment:
This is being a bit OCD, but could we revert the Jupyter notebooks to what is in `develop` and not change the execution numbers for consistency in the documentation?

Reply:
I see, yeah, I'll fix that up sometime today.